scripts/utilities/supported_boards.py

```diff
@@ -4,6 +4,7 @@
 
 
 def path_is_folder_and_has_ioc_file(pathName: str):
+    print(pathName)
```

Comment:
can you remove this print?
also I'm going to delete this file in a follow up PR, really doesn't add any value